packages/examples/trusted-gateway-token/contracts/contracts/TestMaxRetry.sol

```diff
@@ -0,0 +1,47 @@
+//SPDX-License-Identifier: MIT
```

Comment:
Is this file in use? Looks like duplicate

Reply:
the other contracts were in both places, for the consistency I placed this one too.